tone-analyzer/v3.js

```diff
@@ -88,4 +87,24 @@ ToneAnalyzerV3.prototype.tone = function(params, callback) {
   return requestFactory(parameters, callback);
 };
 
+ToneAnalyzerV3.prototype.tone_chat = function(params, callback) {
```

Comment:
@aprilwebster can you add jsdoc here? take a look at https://github.com/watson-developer-cloud/node-sdk/blob/master/personality-insights/v3.js#L45-L59